app/shared/components/Footer/DonationButton/DonationButton.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DonationButton from './DonationButton';
+
+const mockData = {
+  text: 'Donate Now',
+  link: '/donate'
+};
+
+describe('DonationButton', () => {
+  beforeEach(() => {
+    render(<DonationButton data={mockData} />);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('renders button with correct label', () => {
```

Comment:
```suggestion
  it('should render the button with correct label', () => {
```
And do it in all test-cases

Reply:
done